src/theme/index.ts

```diff
@@ -1,44 +1,45 @@
-import * as monaco from 'monaco-editor'
-import { defineVSCodeTheme } from './registry'
-import './semanticHighlight'
-import { IVSCodeTheme } from './tools'
+import { setThemes, setDefaultThemes, IThemeExtensionPoint } from 'vscode/service-override/theme'
+import { Disposable } from 'vscode'
 import defaultThemes from '../languages/extensions/themes.json'
 import themeLoader from '../languages/extensions/themeLoader'
 
-const DEFAULT_VSCODE_THEMES: Partial<Record<string, monaco.editor.BuiltinTheme>> = {
-  'Default Light+': 'vs',
-  'Default Dark+': 'vs-dark',
-  'Default High Contrast': 'hc-black'
-  // 'Default High Contrast Light': 'hc-white' // FIXME: Uncomment me with monaco 0.34 which support hc-light theme
```

Comment:
does it mean we will have the HC light theme now ?

Reply:
Yes, the bug was in the monaco standalone services